README.md

```diff
@@ -1,24 +1,597 @@
+* [![HTML][HTML-shield]][HTML-url]
+* [![JavaScript][JavaScript-shield]][JavaScript-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Gems
+
+#### Deployment
+* [![faraday][gem-faraday]][gem-faraday-url]
+* [![jsonapi-serializer][gem-jsonapi-serializer]][gem-jsonapi-serializer-url]
+
+#### Testing
+* [![debug][gem-debug]][gem-debug-url]
+* [![rspec-rails][gem-rspec-rails]][gem-rspec-rails-url]
+* [![simplecov][gem-simplecov]][gem-simplecov-url]
+* [![factory_bot_rails][gem-factory_bot_rails]][gem-factory_bot_rails-url]
+* [![faker][gem-faker]][gem-faker-url]
+* [![pry][gem-pry]][gem-pry-url]
+* [![shoulda-matchers][gem-shoulda-matchers]][gem-shoulda-matchers-url]
+* [![capybara][gem-capybara]][gem-capybara-url]
+* [![webmock][gem-webmock]][gem-webmock-url]
+* [![vcr][gem-vcr]][gem-vcr-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Database 
+
+<img src="images/schema.png"  width="300" height="800">
+
+### Installation
+
+1. Get an API Key at [INSERT-API-LOCATION-PATH-HERE](INSERT-API-LOCATION-PATH-HERE)
+
+2. Clone the repo
+   ```sh
+   git clone https://github.com/Tavern-Keeper-2308/tavern_keeper_be.git
+   ```
+3. Enter your API
+  Terminal
+   ```sh
+   EDITOR="code --wait" rails credentials:edit
+   ```
+    In editor pop up
+    ```ruby
+    weather_api:
+      key: <YOUR API KEY>
+    ```
+4. Gem Bundle
+   ```sh
+    bundle
+   ```
+5. Rake
+   ```sh
+    rails db:{drop,create,migrate,seed}
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Testing
+#### INSERT TESTING INSTRUCTIONS AND % HERE
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- ENDPOINTS -->
+## Endpoints
+
+### [INSERT-ENDPOINT-NAME-HERE](#INSERT-ENDPOINT-TAG_HERE)
+
+**`ENDPOINT BADGE HERE`**
+##### Example Query/Mutation (MAKE SURE TO DELETE ONE THAT IT NOT IS)
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example GraphQL Variables
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+#### Error Handling
+##### `INSERT-ERROR-CODE-HERE` INSERT-ERROR-EXPLAINATION-HERE
+##### Example Query
+```graphql
+INSERT-QUERY-HERE
+```
+##### Example Response
+```json
+INSERT-JSON-RESPONSE-HERE
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### [Query User](#q-user)
+
+**`ENDPOINT BADGE HERE`**
+##### Example Query
+```graphql
+query getUser($id: Integer!) {
+    queryUsers (byId: $id) {
+        id
+        username
+        password
+        encounters {
+            encounter_id
+        }
+    }
+}
+```
+##### Example GraphQL Variables
+```graphql
+{
+    "id": 55
+}
+```
+##### Example Response
+```json
+{
+    "data": {
+        "queryUsers": [
+            {
+                "'id": 55,
+                "username": "Shrek",
+                "password": "password123"
```

Comment:
I think we're missing a comma here.

Reply:
I also fiddled around with the mock-server stuff on my local Postman, and I don't think the `'` is necessary in front of `id`. It seemed to be working just fine for me without. Did anyone find any info as to why it might be in the starter kit? Or was it maybe just a typo?